sdk/lib/drivers/sound/mmixer/wave.c

```diff
@@ -614,6 +614,46 @@ MMixerGetWaveOutCount(
     return MixerList->WaveOutListCount;
 }
 
+MIXER_STATUS
+MMixerGetWavePosition(
+    _In_ PMIXER_CONTEXT MixerContext,
+    _In_ HANDLE PinHandle,
+    _Out_ DWORD * Position)
+{
+    KSAUDIO_POSITION AudioPosition;
+    KSPROPERTY Property;
+    MIXER_STATUS Status;
+    ULONG Length;
+
+    /* verify mixer context */
+    Status = MMixerVerifyContext(MixerContext);
+
+    if (Status != MM_STATUS_SUCCESS)
+    {
+        /* invalid context passed */
+        return Status;
+    }
```

Comment:
```suggestion
    if (Status != MM_STATUS_SUCCESS)
        return Status;
```
(comment is a bit "trivial")

Reply:
OK, removed the comment.